docs/tempo/website/configuration/_index.md

```diff
@@ -123,7 +123,15 @@ distributor:
 
     # Optional.
     # Enable to log every received trace id to help debug ingestion
-    [log_received_traces: <bool>]
+    # WARNING: Deprecated. Use log_received_spans instead.
+    [log_received_traces: <boolean> | default = false]
+
+    # Optional.
+    # Enable to log every received span to help debug ingestion or calculate span error distributions using the logs
+    log_received_spans:
+        [enabled: <boolean> | default = false]
+        [include_attributes: <boolean> | default = false]
+        [filter_by_status_error: <boolean> | default = false]
 
     # Optional.
     # disables write extension with inactive ingesters. Use this along with ingester.lifecycler.unregister_on_shutdown = true
```

Comment:
"Disables" should be capitalized since it is the first word in a sentence. 
```suggestion
    # Disables write extension with inactive ingesters. Use this along with ingester.lifecycler.unregister_on_shutdown = true
```